src/client/testing/testController/common/server.ts

```diff
@@ -23,7 +25,7 @@ export class PythonTestServer implements ITestServer, Disposable {
 
     public port: number;
 
-    constructor(private executionFactory: IPythonExecutionFactory) {
+    constructor(private executionFactory: IPythonExecutionFactory, private debugLauncher?: ITestDebugLauncher) {
```

Comment:
```suggestion
    constructor(private executionFactory: IPythonExecutionFactory, private debugLauncher: ITestDebugLauncher) {
```

debug launcher should not be optional.